app/jobs/regular/group_smtp_email.rb

```diff
@@ -40,9 +40,12 @@ def execute(args)
         return skip(email, post, recipient_user, :group_smtp_topic_deleted)
       end
 
-      if !group.smtp_enabled
-        return skip(email, post, recipient_user, :group_smtp_disabled_for_group)
-      end
+      email = args[:email]
+      cc_addresses = args[:cc_emails].map do |cc|
```

Comment:
This is the main change, I just moved around some other code while I was at it, no actual material effect will be made by moving that code.